ext/encryptors/rotn/rotn_encrypt.c

```diff
@@ -64,6 +64,9 @@
  * In this case, we transform all bytes in the input.
  */
 
+/* Add u_char type for windows compatibility */
+typedef unsigned char   u_char;
+
```

Comment:
@daveh86, probably it's better to replace all uses of u_char by uint8_t, which should already be defined (it's in stdint.h).